p/mobil-nrw/products.js

```diff
@@ -67,7 +67,7 @@ const products = [
 	},
 	{
 		id: 'ec-ic',
-		mode: 'ec-ic',
+		mode: 'train',
```

Comment:
Even though `ec-ic` was not intended, this could be considered a breaking change by some people who depend on this specific value. I'm still fine with publishing with a non-major version though.